### ISSUE TITLE
I need help on Node management

### ISSUE THREAD

**Issue by blacbrue** [fillerInk/this-video-has-x-views#2]
DO we need a VPS to handle the node requests or something else will handle it?

**Comment by sreeram-venkitesh** [comment]
No, we don't need a VPS. I hosted the code on a [Deta Micro](https://docs.deta.sh/docs/micros/about). I have made a video tutorial on how I did it [here](https://www.youtube.com/watch?v=QwecvVvESVU)
